examples/pysdl2.py

```diff
@@ -0,0 +1,218 @@
+"""
+ Simple SDL2 / cefpython3 example.
+
+ Only handles mouse events but could be extended to handle others.
+
+ Requires pysdl2 (and SDL2 library).
+
+ Install instructions.
+ 
+ 1. Install SDL libraries for your OS, e.g. for Fedora:
+
+   dnf install SDL2 SDL2_ttf SDL2_image SDL2_gfx SDL2_mixer
+
+ 2. Install PySDL via PIP:
+
+   pip2 install PySDL2
+
+ Tested configurations:
+ - SDL2 2.0.5 with PySDL2 0.9.3 on Fedora 25 (x86_64)
+"""
+
+import os
+import sys
+from cefpython3 import cefpython as cef
+try:
+    import sdl2
+    import sdl2.ext
+except ImportError:
+    print "SDL2 module not found - please install"
+    sys.exit(1)
+try:
+    from PIL import Image
+except ImportError:
+    print "PIL module not found - please install"
+    sys.exit(1)
+
+class LoadHandler(object):
+    """Simple handler for loading URLs."""
+    
+    def OnLoadingStateChange(self, browser, is_loading, **_):
+        if not is_loading:
+            print "loading complete"
```

Comment:
Are you able to test it using Python3? If not, not a big issue I can do this, just fix the most obvious issues like the print statement.